tests/unit/lms/services/course_test.py

```diff
@@ -39,6 +40,7 @@ def test_it_does_nothing_if_theres_already_a_matching_row(
         existing_course = pyramid_request.db.query(Course).one()
         assert not existing_course.settings.get("canvas", "sections_enabled")
 
+    @pytest.mark.usefixtures("application_instance")
```

Comment:
As a result of the above, we actually _do_ use the `application_instance` directly or indirectly in every test. It looks like `test_any_with_setting()` below doesn't, but it actually does by virtue of the `add_courses_with_settings` fixture.

So we could put back the `autouse=True` and remove these lines.